### DISCUSSION TITLE
Is there a PowerShell commandlet equivalent of `winget list`?

### DISCUSSION THREAD

**Discussion by RalfReddings** [microsoft/winget-cli#4106] [Q&A] [answered]
I have been messing around with the pre release module for Winget
Looking at the commandlet list:
    
    CmdletsToExport = @(
     'Get-WinGetVersion'
     'Find-WinGetPackage'
     'Get-WinGetPackage'
     'Get-WinGetSource'
     'Install-WinGetPackage'
     'Uninstall-WinGetPackage'
     'Update-WinGetPackage'
     'Get-WinGetUserSettings'
     'Set-WinGetUserSettings'
     'Test-WinGetUserSettings'
     'Assert-WinGetPackageManager'
     'Repair-WinGetPackageManager'
     'Enable-WinGetSetting'
     'Disable-WinGetSetting'
     'Get-WinGetSettings'
     'Add-WinGetSource'
     'Remove-WinGetSource'
     'Reset-WinGetSource'
    )
    
There does not seem to be a equivalent commandlet for `winget list`.

I am using the module version `0.2.2` which I installed with `Install-Module -Name Microsoft.WinGet.Client -RequiredVersion 0.2.2`

Any help would be greatly appreciated!

**Comment by mdanish-kh** [comment, marked as answer]
It's `Get-WinGetPackage`

**Reply by RalfReddings** [reply]
Aaah that where it was. That makes sense. Thank you.
